docker-images/zrok-private-access/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '3'
+services:
+  zrok-enable-init:
+    image: busybox
+    # matches uid:gid of "nobody" in zrok container image
+    command: chown -Rc 65534:65534 /mnt/.zrok
+    user: root
+    volumes:
+      - zrok_env:/mnt/.zrok
+  zrok-enable:
+    image: docker.io/openziti/zrok:enable-headless  # FIXME: resume :latest if >= :0.3.2
+    depends_on:
+      zrok-enable-init:
+        condition: service_completed_successfully
+    entrypoint:
+      - bash
+      - -c
+      - |
+        if [[ -s /mnt/.zrok/environment.json ]]; then
+          echo "INFO: noop: zrok environment is already enabled"
+          exit 0
+        else
+          echo "INFO: running: zrok $$(sed -E 's/\S{12,}/************/' <<< $${@})"
+          exec zrok $${@}
+        fi
+    command: -- enable --headless ${ZROK_ENABLE_TOKEN}
+    volumes:
+      - zrok_env:/mnt/.zrok
+    environment:
+      HOME: /mnt
+      ZROK_ENABLE_TOKEN:
+      ZROK_API_ENDPOINT: https://api.zrok.io/
+  zrok-private-access:
+    image: docker.io/openziti/zrok
+    command: access private --bind 0.0.0.0:9191 ${ZROK_ACCESS_TOKEN}
```

Comment:
0.0.0.0 should come from the environment imo. not default. is that easy/hard? if easy can you do that?

Reply:
It has to be 0.0.0.0 in this case unless we do some Docker API gymnastics with a custom image to discover the container IP where it should bind, which is the only IP it will bind anyway with 0.0.0.0 other than the loopback.